afl-cmin

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env sh
 export AFL_QUIET=1
-export ASAN_OPTIONS=detect_leaks=0
```

Comment:
why did you remove this? otherwise targets compiled with asan produce errors.
that an asan compiled showmap does not have leaks we can test with alf-showmap directly

Reply:
How do you mean? We don't care about targets here as this is the aflpp makefile?

Reply:
we dont care about leaks in targets is what I means :) and with your removal you enable printing+erroring on them.
its in the afl-cmin script

Reply:
ooh oops that was a fail :)